packages/xstate-fsm/src/index.ts

```diff
@@ -86,6 +86,15 @@ export function createMachine<
     actions?: StateMachine.ActionMap<TContext, TEvent>;
   } = {}
 ): StateMachine.Machine<TContext, TEvent, TState> {
+  
+  Object
```

Comment:
Can you wrap this in `if (!IS_PRODUCTION)`?

Also, the documentation does say that it does not support nested states.